src/routes/+layout.svelte

```diff
@@ -18,6 +19,15 @@
 
 <svelte:head>
 	<title>{data.websiteTitle}</title>
+	{#if data.roleId !== CUSTOMER_ROLE_ID && data.viewportFor === 'employee'}
+		<meta name="viewport" content="width=device-width" />
+	{:else if data.roleId === CUSTOMER_ROLE_ID && data.viewportFor === 'visitors'}
+		<meta name="viewport" content="width=device-width" />
+	{:else if data.viewportFor === 'everyone'}
+		<meta name="viewport" content="width=device-width" />
+	{:else}
+		<meta name="viewport" content="width={data.viewportContentWidth}" />
+	{/if}
```

Comment:
Let's move this to the backend, the backend should  send `viewportWidth` depending on the role.

Here we should only have : 
```suggestion
	<meta name="viewport" content="width={data.viewportWidth}" />
```

This means that you will need to send both `viewportFor` and `viewportContentWidth` in the admin's `+page.server.ts`